### ISSUE TITLE
Grab Balloon buttons missing on-demand help text.

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#302]
The latest version of BASE seems to be missing the grab button on-demand help text:

"Get ready to move balloon. Once grabbed, press W, A, S, or D key to move up, left, down, or right. Space to release."

This content provides more detail than the interaction in in the grab alert.

The HTML should look similar to other button/help-text patterns:
```html
<button>Grab Yellow Balloon</button>
<p>Get ready to move balloon. Once grabbed, press W, A, S, or D key to move up, left, down, or right. Space to release.</p>
```

**Comment by jessegreenberg** [comment]
Hmm, sorry @terracoda looking into it.

**Comment by jessegreenberg** [comment]
Fixed this issue in the above commits, here is what it looks like now.
![capture](https://user-images.githubusercontent.com/6396244/30744903-5c11cf9c-9f61-11e7-8fa4-f8bc4180a715.PNG)

**Comment by terracoda** [comment]
@jessegreenberg, thanks! Looks good.
And no need to apologize. Your work is awesome!
